src/System.Buffers.Primitives/System/Buffers/OwnedPinnedArray.cs

```diff
@@ -6,37 +6,6 @@
 
 namespace System.Buffers
 {
-    public class OwnedNativeMemory : OwnedMemory<byte>
```

Comment:
I misunderstood a comment you made when we were chatting. Thought you needed those types. I'll move them back.